### PR TITLE
Change billing

### PR DESCRIPTION
**Pull request by syanaga**
## Description & motivation

Changing the description how to change billing info to match the new UI (from the UI Refresh)

## Checklist

If you removed existing pages (delete if not applicable):

### CODE REVIEW

**Comment by syanaga** [comment]
@runleonarun this PR is ready for review :)

File: website/docs/faqs/Accounts/change-billing.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ sidebar_label: 'How to update billing'
 id: change-billing
 ---
 
-If you'd like to change card details, the account owner can do this by going to their Account Setting -> Billing page.
+If you would like to change your account's credit card details, the account owner can do this by clicking the gear icon in the upper right corner and going to Account Settings  &#8594; Billing  &#8594; Payment Information. From there, click the Edit button in the upper right corner of the Payment Information section to input the new credit card details.
```
Comment (nghi-ly) [inline website/docs/faqs/Accounts/change-billing.md]:
```suggestion
If you want to change your account's credit card details, select the gear menu in the upper right corner of dbt Cloud. Go to Account Settings  &#8594; Billing  &#8594; Payment Information. In the upper right corner of **Payment Information**, click **Edit**  to enter the new credit card details. Only the _account owner_ can make this change. 
```

Comment (nghi-ly) [reply website/docs/faqs/Accounts/change-billing.md]: suggest breaking up the first sentence a tiny bit to help increase readability and for faster reading

**Review by nghi-ly — CHANGES_REQUESTED** [review]
thanks for making the change! left a sugg to help improve readability and with navigation

**Comment by nghi-ly** [comment]
stephanie 👍 the sugg in slack: 

Screen Shot 2022-07-26 at 3 51 28 PM

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**